source/agora/node/Validator.d

```diff
@@ -585,9 +585,9 @@ public class Validator : FullNode, API
             return Enrollment.init;
         }
 
-        log.trace("Sending Enrollment at height {} for {} cycles with {}",
-            next_height, this.params.ValidatorCycle, enroll_key);
         const enrollment = this.enroll_man.createEnrollment(enroll_key, avail_height);
+        log.trace("Sending Enrollment to be stored in pools for enrolling at height {} to participate in blocks {} to {}",
```

Comment:
`pool` seems right, not `pools`. Anyway Isn't the shorter message better, like "Sending Enrollment enrolling at height" ?

Reply:
I dropped the mention of pools and reduced the length of the message.